### DISCUSSION TITLE
Zaaktype aanpassen en wijzigingen toepassen op lopende zaken

### DISCUSSION THREAD

**Discussion by OlavAllema** [open-zaak/open-zaak#1578] [General]
In de praktijk ervaren we dat er behoefte is aan het toepassen van wijzigingen aan een zaaktype en die toe te passen voor lopende zaken. Een nieuwe versie van een zaaktype is nu niet geldig voor lopende zaken. 
Hoe staan jullie t.o.v. deze casus? Is dit een aanpassing die in Open Zaak zou moeten komen? Of zou deze behoefte ergens anders geregeld moeten worden?

**Comment by Michel-Verhoeven** [comment]
Wij lopen er in de praktijk ook tegenaan dat we een zaaktype voor een lopende zaak willen wijzigen. De huidige api standaard verplicht de validatie ook niet meer https://vng-realisatie.github.io/gemma-zaken/standaard/zaken/redoc-1.5.1.html#tag/zaken/operation/zaak_partial_update (ook concept 1.6.0 niet) @joeribekker wat denk jij hiervan, kan het aangepast worden?
